docs/source/guides/user/chapters/installing.rst

```diff
@@ -56,7 +56,7 @@ version 29.  To subscribe to the latest version stream, run::
 
 Or, to use the LTS (Long Term Stability) version stream, run::
 
-  $ dnf module enable avocado:82lts
+  $ dnf module enable avocado:103lts
```

Comment:
This doesn't exist yet, but it's accurate.  We'll have that stream created when the time comes.

Reply:
We can change it with the release itself, but I think it is not necessary. It is preparation for 103lts therefore IMO we will be fine to have it mentioned few weeks before the release.